### DISCUSSION TITLE
The Out Of Box Experience

### DISCUSSION THREAD

**Discussion by Pixel556** [blueedgetechno/win11React#352] [Q&A] [answered]
### The OOBE
It would be awesome if you added the OOBE. Make it so that the blog is another app and the Get started app leads you right to the OOBE without wiping anything.

**Comment by luni-moon** [comment, marked as answer]
> ### The OOBE
> 
> It would be awesome if you added the OOBE. Make it so that the blog is another app and the Get started app leads you right to the OOBE without wiping anything.

Please look at past issues and discussions, when posting
either one. (#302)

**Comment by yashash-pugalia** [comment]
We now have the oobe just open the Get Started app from the start menu.
